apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -113,13 +113,7 @@ static void startAppCenter(Application application, String startTypeString) {
         int savedTrack = sSharedPreferences.getInt(application.getString(R.string.appcenter_distribute_track_state_key), 0);
         if (savedTrack != 0) {
             try {
```

Comment:
Please remove try {... } catch(..) around the call

Reply:
@annakocheshkova Yes, I have removed in new commit.